### PR TITLE
Cl con 448 fixing ie class issue

### PR DESCRIPTION
**Pull request by carynligon**
#### Overview
Changed classLists to set and getAttribute
-
- Closes #

#### Todo

-

#### Screenshots/Videos (User Facing Changes)

> _Insert responsive image(s) and/or video(s)_

### CODE REVIEW

File: src/components/Trend/Trend.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Trend extends Component {
 
   componentDidMount() {
     const path = document.querySelector('.trend-line path');
```
Comment (abiodun0) [inline src/components/Trend/Trend.js]: To make this a little bit easier. and not to break the compositional nature of react components. I think we can have this condition in the render method and add the class conditional.

The reason for this is using reference to DOM node breaks the compositional nature of react. We shouldn't use dom reference to mutate the nodes that are already being controlled by virtual DOM. that way it wouldn't be confused on what to update and what not. Does this make sense?

File: src/components/Trend/Trend.js [hunk 2]
```diff
@@ -63,13 +63,13 @@ class Trend extends Component {
     const { autoDraw, autoDrawDuration, autoDrawEasing } = this.props;
 
     const path = document.querySelector('.trend-line path');
-    path.classList.add('animate');
+    path.setAttribute('class', 'trend-line animate');
```
Comment (abiodun0) [inline src/components/Trend/Trend.js]: I believe we have polyfills for doms that works well with using classList. If that's not available we can easily copy the polyfills from mozillas documentation(https://gist.github.com/k-gun/c2ea7c49edf7b757fe9561ba37cb19ca) with the source here and add it to the source's library so it's easily accessible.

Class lists are known to be faster that there get and setAttributes aliases

**Review by abiodun0 — CHANGES_REQUESTED** [review]
Great work. Just some comments about the compositional nature of react and how this implementation might break it and cause unknown bugs for the codes we're writing.. hope this makes sense?

File: src/components/Trend/Trend.js [hunk 3]
```diff
@@ -62,17 +61,14 @@ class Trend extends Component {
   autoDraw(first) {
     const { autoDraw, autoDrawDuration, autoDrawEasing } = this.props;
 
-    const path = document.querySelector('.trend-line path');
-    path.classList.add('animate');
-    // remove animate class after animation duration
+    // const path = document.querySelector('.trend-line path');
```
Comment (abiodun0) [inline src/components/Trend/Trend.js]: Can we remove commented out code too?

**Review by abiodun0 — APPROVED** [review]
Great job. Lets remove commented out code too

**Review by ghost — APPROVED** [review]
Great Job

**Pull request merged**